backtrace-library/src/main/java/backtraceio/library/metrics/SummedEventsPayload.java

```diff
@@ -0,0 +1,20 @@
+package backtraceio.library.metrics;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+public class SummedEventsPayload extends EventsPayload<SummedEvent> {
+    @SerializedName("summed_events")
+    private final ConcurrentLinkedDeque<SummedEvent> summedEvents;
+
+    protected SummedEventsPayload(ConcurrentLinkedDeque<SummedEvent> events, String application, String appVersion, int droppedEvents) {
```

Comment:
do we need to have this last parameter if it is always zero? https://github.com/backtrace-labs/backtrace-android/pull/32/files#diff-4c63f676e3e4590948f173277d9fb78f972665f4aae8f1e3ee2b4aea64e85a80R42 this is only place where we call this constructor

Reply:
same for UniqueEventsPayload

Reply:
Ah that's a good point, thank you!